packages/backend/src/asset/service.ts

```diff
@@ -197,3 +221,13 @@ async function getAssetsPage(
 async function getAll(deps: ServiceDependencies): Promise<Asset[]> {
   return await Asset.query(deps.knex).whereNull('deletedAt')
 }
+
+async function setAssetOn(
```

Comment:
Actually, we don't need to change the model at all, since `asset` is not readonly. I'm mostly coming from the approach that this isn't assetService's responsibility.

We can just do something like 
```ts
incomingPayment.asset = await assetService.get(incomingPayment.assetId) as Asset
```
directly. And if you want to avoid using `as`, we can just have a `assetService.getOrThrow` that returns `Promise<Asset>`.

Reply:
Updated.